tests/integrated/test-thumbnail.R

```diff
@@ -0,0 +1,88 @@
+test_that("set_thumbnail works with local images", {
+  scoped_experimental_silence()
+  img_path <- rprojroot::find_package_root_file("tests/testthat/examples/logo.png")
+
+  res <- set_thumbnail(cont1_content, img_path)
+
+  expect_true(validate_R6_class(res, "Content"))
+})
+
+test_that("get_thumbnail works", {
+  scoped_experimental_silence()
+  img_path <- rprojroot::find_package_root_file("tests/testthat/examples/logo.png")
+
+  tmp_img <- fs::file_temp(pattern = "img", ext = ".png")
```

Comment:
I know this is in testing code only, but do we need `fs` here? It would be good not to add on to places we'll need to remove it later.

Reply:
Lol, I had to do a `git blame` to figure out what happened here. These were existing `*_image` tests that I had moved to their own file a little while back, but apart from switching them over to use the new functions, I don't think I changed anything, hence the reference to `fs`.

Reply:
You can see the block that vanished from `test-deploy.R` to see where they came from.